rust-toolchain

```diff
@@ -1 +0,0 @@
-1.44.1
```

Comment:
How come we're deleting this?

Reply:
I guess it's unusual for a library to specify the toolchain.  It should work with a decent spread of recent rust versions.

Reply:
As far as I understand, rust-toolchain is not supposed to be checked in in a library crate.

Reply:
Unless you use a nightly toolchain